cartography/intel/aws/rds.py

```diff
@@ -185,4 +274,4 @@ def sync_rds_instances(neo4j_session, boto3_session, regions, current_aws_accoun
         logger.info("Syncing RDS for region '%s' in account '%s'.", region, current_aws_account_id)
         data = get_rds_instance_data(boto3_session, region)
         load_rds_instances(neo4j_session, data, region, current_aws_account_id, aws_update_tag)
-    cleanup_rds_instances(neo4j_session, common_job_parameters)
+        cleanup_rds_instances_and_db_subnet_groups(neo4j_session, common_job_parameters)
```

Comment:
Was the indentation change intentional? Why?

Reply:
good catch thanks